js/base-tool.js

```diff
@@ -549,6 +549,163 @@ function baseTool () {
+						<tr>
+							<td><input type="checkbox" id="apply_dynamic_lighting_enabled" name="apply_dynamic_lighting_enabled"></td>
+							<td><label for="dynamic_lighting_enabled">Dynamic Lighting Enabled:</label></td>
+							<td><input type="checkbox" id="dynamic_lighting_enabled" name="dynamic_lighting_enabled"></td>
+						</tr>
+						<tr>
+							<td><input type="checkbox" id="apply_daylight_mode_enabled" name="apply_daylight_mode_enabled"></td>
+							<td><label for="daylight_mode_enabled">Daylight mode:</label></td>
+							<td><input type="checkbox" id="daylight_mode_enabled" name="daylight_mode_enabled"></td>
+						</tr>
+						<tr>
+							<td><input type="checkbox" id="apply_lightupdatedrop" name="apply_lightupdatedrop"></td>
+							<td><label for="lightupdatedrop">Light Update Drop:</label></td>
+							<td><input type="checkbox" id="lightupdatedrop" name="lightupdatedrop"></td>
+						</tr>
+					</tbody>
+				</table>
+                <hr>
+				<p style="text-align: right;"><button class="btn btn-danger modifier">Apply</button></p>
+				</div>
+				`,
+			dialogFn: () => {
+				$("#d20plus-mass-page-modify").dialog({
+					autoOpen: false,
+					resizable: true,
+					width: 600,
+					height: 800,
+				});
+			},
+			openFn: () => {
+				function modifyPage (model, changes) {
+					changes.forEach(change => { model.attributes[change.label] = change.value
+					});
+					model.save()
+				}
+
+				const $win = $("#d20plus-mass-page-modify");
+				$win.dialog("open");
+
+				const $lst = $win.find(`.list`).empty();
+
+				d20.Campaign.pages.models.forEach(m => {
+					$lst.append(`
+							<label class="import-cb-label import-cb-label--img" data-listid="${m.id}">
+								<input type="checkbox">
+								<img class="import-label__img" src="${m.attributes.thumbnail}">
+								<span class="name col-9">${m.attributes.name}</span>
+								<span style="display: none;" class="page-id">${m.id}</span>
+							</label>
+						`);
+				});
+
+				const pageList = new List("modify-pages-list", {
+					valueNames: ["name", "page-id"],
+				});
+
+				const $cbAll = $win.find(`.select-all`).off("click").click(() => {
+					pageList.items.forEach(it => {
+						$(it.elm).find(`input[type="checkbox"]`).prop("checked", $cbAll.prop("checked"));
+					});
+				});
+
+				function collectCheckedInputs () {
+					const table = document.getElementById("modifications-table");
+					const rows = table.querySelectorAll("tbody tr");
+					const checkedInputs = [];
+
+					rows.forEach(row => {
+						const applyCheckbox = row.querySelector("td:first-child input[type=\"checkbox\"]");
+						if (applyCheckbox.checked) {
+							const input = row.querySelector(`#${applyCheckbox.id.slice("apply_".length)}`);
+							const inputName = input.id;
+							let inputValue = ""
+
+							if (input.type === "checkbox")
+							{
+								inputValue = input.checked
+							}
+							else {
+								inputValue = input.options[input.selectedIndex].value
+							}
```

Comment:
easy on the line-count there

```suggestion
							if (input.type === "checkbox") inputValue = input.checked;
							else inputValue = input.options[input.selectedIndex].value;
```